docs/products/clickhouse/howto/fetch-query-statistics.rst

```diff
@@ -0,0 +1,28 @@
+Fetch query statistics for Aiven for ClickHouse®
+================================================
+
+About fetching query statistics
+-------------------------------
+
+Usually, query statistics in ClickHouse can be obtained using the `system.query_log` table, which stores statistics of each executed query, including memory usage and duration. In Aiven for ClickHouse®, the `system.query_log table` is currently not accessible for the purpose of obtaining query statistics.
```

Comment:
For ``system.query_log``, please use the double quotes to get the code font (single quotes you used give italics). Sorry for being not clear enough about that!

Reply:
My bad! i was not aware about the double quotes. ( good to know ). Thanks!